{{cookiecutter.project_name}}/docs/license.md

```diff
@@ -0,0 +1,7 @@
+# License
+
+```{literalinclude} ../LICENSE
+---
+language: none
```

Comment:
Small detail...is this required or can we just do it all on one line, no YAML options passed?

Reply:
For this case, we can rewrite it as:

\```{literalinclude} ../LICENSE 
\```

But we can't just do it all no one line, like this \```{literalinclude} ../LICENSE\```  it will not render ^ ^


Also, yaml options can be replaced with: 


\```{literalinclude} ../LICENSE
:language: none
\```